src/pages/about.tsx

```diff
@@ -63,7 +63,17 @@ const AboutPage: React.FC<PageProps> = () => {
             A list of some of our active members (in alphabetical order):{' '}
             {formatter.format(membersNames)}
           </p>
-          <p className='mt-4'>Want your name to be a part of this list? Make a pull request!</p>
+          <p className='mt-4'>
+            Want your name to be a part of this list? Make a{' '}
+            <a
+              className='text-purple-400 font-semibold'
```

Comment:
Small thing, but I think `text-purple-400` doesn't provide enough contrast against the white background. Maybe `text-purple-600` or `text-purple-500` would be better here.

Reply:
Changed to `text-purple-600`